polarssl/Makefile

```
@@ -0,0 +1,37 @@
+# Port Metadata
+PORTNAME =          polarssl
+PORTVERSION =       1.3.9
+
+MAINTAINER =        Damian Parrino <bucanero@users.noreply.github.com>
+LICENSE =           GPL2
+SHORT_DESC =        A Secure Socket Layer library.
+
+# This port uses CMake
+PORT_BUILD =        cmake
+
+# What files we need to download, and where from.
+DOWNLOAD_SITE  =    https://src.fedoraproject.org/repo/pkgs/polarssl/polarssl-1.3.9-gpl.tgz/48af7d1f0d5de512cbd6dacf5407884c
+DOWNLOAD_FILES =    polarssl-1.3.9-gpl.tgz
+
+TARGET =            libpolarssl.a
+
+# Library headers
+HDR_DIRECTORY =     include/polarssl
+
+# Set cmake build settings
+CMAKE_ARGS =        -DCMAKE_BUILD_TYPE=Release -DENABLE_TESTING=FALSE -DENABLE_PROGRAMS=FALSE
```

```suggestion
CMAKE_ARGS =        -DCMAKE_BUILD_TYPE=Release -DENABLE_TESTING=FALSE -DENABLE_PROGRAMS=FALSE -DHAVE_MACHINE_ENDIAN_H
```
By sending the endian flag like this, at least that piece of the patch is unneeded.

---

I've tested adding that flag, but I'm getting an error from CMake:

```
CMake Error: Parse error in command line argument: HAVE_MACHINE_ENDIAN_H
 Should be: VAR:type=value
 ```

---

also, if I use `-DHAVE_MACHINE_ENDIAN_H=1` instead, I get:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_INSTALL_INCLUDEDIR
    CMAKE_INSTALL_LIBDIR
    HAVE_MACHINE_ENDIAN_H
```

and then it fails when building `net.c`:
```
/opt/toolchains/dc/kos-ports/polarssl/build/polarssl-1.3.9/library/net.c:91:10: fatal error: endian.h: No such file or directory
   91 | #include <endian.h>
      |          ^~~~~~~~~~
compilation terminated.
```